README.md

```diff
@@ -319,77 +322,204 @@ spec:
 
     ```bash
     kubectl get secrets
     NAME         TYPE     DATA   AGE
-    my-binding   Opaque   5      32s
+    my-secret   Opaque   5      32s
     ```
     
     See [Using Secrets](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets) to learn about different options on how to use the credentials from your application running in the Kubernetes cluster, 
 
 [Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
 
-##### Formats of Output Secret
-
-###### Key- Value Pairs (Default)
-The binding object includes credentials returned from the broker and service instance info presented as key-value pairs.
-```bash
-#Credentials
-uri: https://my-service.authentication.eu10.hana.ondemand.com
-username: admin
-password: ********
-
-#Service instance info
-instance_guid: <instance_guid> // The service instance ID
-instance_name: my-service-btp-name // Taken from the service instance external_name field if set. Otherwise from metadata.name
-plan: sample-plan // The service plan name                
-type: sample-service  // The service offering name
+##### Secret Formats
+
+You can use different attributes in your `ServiceBinding` resource for the purpose of generating different formats of your `Secret` resources.
+
+Even though `Secret` resources can come in various formats, they all share a common basic content. The parameters within the `Secret` fall into two categories:
+
+- Credentials returned from the broker: These credentials allow your applications to connect to the `ServiceInstance`.
+- Attributes of the associated `ServiceInstance`: This information provides details about the service instance itself.
+  
+Now let's explore these various formats:
+
+###### Key-Value Pairs (Default)
+
+Input:
+
+```yaml
+apiVersion: services.cloud.sap.com/v1
+kind: ServiceBinding
+metadata:
+  name: sample-binding
+spec:
+  serviceInstanceName: sample-instance
+```
+Output:
+
+```yaml
+apiVersion: v1
+kind: Secret
+stringData:
+  uri: https://my-service.authentication.eu10.hana.ondemand.com
+  client_id: admin
+  client_secret: ********
+  instance_guid: your-sample-instance-guid // The service instance ID
+  instance_name: sample-instance // Taken from the service instance external_name field if set. Otherwise from metadata.name
+  plan: sample-plan // The service plan name                
+  type: sample-service  // The service offering name
 ```
 
 ###### Credentials as JSON Object
-To show credentials returned from the broker as a JSON object, use the 'secretKey' attribute in the service binding spec.
 
-The value of 'secretKey' is the name of the key that stores the credentials in JSON format.
+To show credentials returned from the broker within the `Secret` resource  as a JSON object, use the 'secretKey' attribute in the `ServiceBinding` spec.
+The value of this 'secretKey' must be the name of the key that stores the credentials in JSON format:
```

Comment:
not "must be"

e.g., The value of 'secretKey' would the name of a key which stores the credentials in JSON format